### PR TITLE
qa/workunits/rbd: fix

### PR DESCRIPTION
**Pull request by liewegas**

### CODE REVIEW

**Comment by liewegas** [comment]
http://pulpito.ceph.com/sage-2016-12-15_19:00:01-rbd-wip-sage-testing---basic-smithi/

File: qa/workunits/rbd/rbd_mirror.sh [hunk 1]
```diff
@@ -7,12 +7,7 @@
 # socket, temporary files, and launches rbd-mirror daemon.
 #
 
-if [ -n "${CEPH_REF}" ]; then
-  wget -O rbd_mirror_helpers.sh "https://git.ceph.com/?p=ceph.git;a=blob_plain;hb=$CEPH_REF;f=qa/workunits/rbd/rbd_mirror_helpers.sh"
-  . ./rbd_mirror_helpers.sh
-else
-  . $(dirname $0)/rbd_mirror_helpers.sh
-fi
+. ./rbd_mirror_helpers.sh
```
Comment (trociny) [inline qa/workunits/rbd/rbd_mirror.sh]: @liewegas I suppose it wont work when running this locally from cmake build dir, i.e.:
```
../qa/workunits/rbd/rbd_mirror.sh
```

Comment (liewegas) [inline qa/workunits/rbd/rbd_mirror.sh]: oh right.  i'll do dirname unconditionally

**Comment by smithfarm** [comment]
@liewegas Are we going to try to backport these changes to jewel?

**Comment by dillaman** [comment]
@smithfarm I think this will need to get backported to all active "testing" branches (including possibly some EoLed releases due to upgrade tests).

**Comment by liewegas** [comment]
Yep!  I was just going to make sure they passed first.  But either way 
we'll backport these (+ fixes) so go ahead!

**Comment by liewegas** [comment]
jewel and hammer

**Comment by smithfarm** [comment]
OK, created the master and backport tracker issues.

**Pull request merged**